src/main/scala/org/scalacheck/util/Pretty.scala

```diff
@@ -115,7 +115,7 @@ object Pretty {
   }
 
   implicit def prettyThrowable(e: Throwable): Pretty = Pretty { prms =>
-    val strs = e.getStackTrace.map { st =>
+    val strs = e.getStackTrace.map(_.nn).map { st =>
```

Comment:
Java method returns array of type `Array[Null | T]`. We need to map it to `Array[T]`.